ui/app/AppLayouts/Profile/stores/AdvancedStore.qml

```diff
@@ -32,6 +32,16 @@ QtObject {
         readonly property string keycard: "keycard"
     }
 
+    function setGlobalNetworkId() {
```

Comment:
@jrainville personally I don't like this mess here in `AdvancedStore`, why don't we be uniform and do it in the same way we did for `mainModule`:

- add this to `Global`:
```
property var advancedModuleInst
property string currentNetworkId: ""
onAdvancedModuleInstChanged: {
    currentNetworkId = advancedModuleInst.currentNetworkId;
}
```
- there is `onAppStateChanged` slot in `ui/main.qml` and there you just add `Global.advancedModuleInst = profileSectionModule.advancedModule`

That's all.

Reply:
`Global` is not a store hence it shouldn't keep any instance of the backend, same for `main.qml`, shouldn't directly use `profileSectionModule.advancedModule`. I don't see any problem of this happening here which aligns well with the structure we attempt to maintain in QML.

Reply:
So is it better to leave it as is or change it like Sale recommended. I personally don't care that much

Reply:
It's as is, as it's not introducing any regressions